examples/typescript-simplechat/readme.md

```diff
@@ -1,14 +1,10 @@
 # Simple Chat Example
```

Comment:
A new name for this one could just be `typescript-chat`

Reply:
we have simple generate, because its almost the simplest possible thing. folks should know whats the absolute easiest way to start.